app/controllers/cards_controller.rb

```diff
@@ -43,6 +43,15 @@ def destroy
     redirect_to cards_path
   end
 
+  def test
+     @card = Card.cards_older_today.first
+  end
+
+  def proverka
+    render 'cards/test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.